reana-ui/src/config.js

```diff
@@ -11,8 +11,8 @@
 const config = {
   name: "CERN REANA UI",
   api: "https://reana.cern.ch",
-  poolingSecs: 5,
-  docsURL: "https://reana.readthedocs.io/en/latest/",
+  poolingSecs: 15,
+  docsURL: "https://docs.reana.io/",
```

Comment:
Better use HTTP only for `docs.reana.io` still, until we have proper SSL certificate